lib/sidekiq/cloudwatchmetrics.rb

```diff
@@ -189,7 +189,7 @@ def publish
           unless process_utilization.nan?
             process_dimensions = [{name: "Hostname", value: process["hostname"]}]
 
-            if process["tag"]
+            unless process['tag'].empty?
```

Comment:
Is it possible that `process["tag"]` is `nil`?
```ruby
if process["tag"] && !process["tag"].empty?
```

(Please keep in mind I'm a random dev and not the repository maintainer.)

Reply:
No, for those apps which are Rails based
Yes, for those apps which are only Ruby based